src/components/info/location-info.vue

```diff
@@ -0,0 +1,193 @@
+  if (location) {
+    const info = await queryInfo(location, map.getView().getProjection())
+    shortUrl.value = info.shortUrl
+    qrUrl.value = getQRUrl(info.shortUrl)
+    clickCoordinateLuref.value = info.clickCoordinateLuref
+    formatted_coordinates.value = info.formatted_coordinates
+    elevation.value = await getElevation(info.clickCoordinateLuref)
+    address.value = await getNearestAddress(info.clickCoordinateLuref)
+  }
+}
+
+const isRapportForageVirtuelAvailable = computed(() => {
+  const userRole = 'ACT'
+  return userRole === 'ACT'
+})
+const downloadingRepport = ref(false)
+const isInBoxOfLidar = computed(() => true)
+const isCyclomediaAvailable = computed(() => true)
+const isImagesObliquesAvailable = computed(() => true)
+
+const getLidarUrl = () => 'bla'
+const cyclomediaUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `http://streetsmart.cyclomedia.com/streetsmart?q=${clickCoordinateLuref.value[0]};${clickCoordinateLuref.value[1]}`
+    : ''
+)
+const imagesObliquesUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `https://oblique.geoportail.lu/publication/viewer?x=${clickCoordinateLuref.value[0]}&y=${clickCoordinateLuref.value[1]}&crs=2169`
+    : ''
+)
+
+const addRoutePoint = () => '// TODO: add point to route'
+
+const open = ref(true)
+const { isStreetviewActive } = storeToRefs(useInfoStore())
+const toggleStreetview = () => {
+  isStreetviewActive.value = !isStreetviewActive.value
+}
+
+function downloadRapportForageVirtuel() {
+  downloadingRepport.value = true
+  setTimeout(() => (downloadingRepport.value = false), 2000)
+}
+</script>
+
+<template>
+  <div class="flex flex-row">
+    <div class="grow-[2] flex flex-col content-end">
+      <div class="grow"></div>
+      <h3 class="text-3xl text-white">
+        {{ t('Short Url', { ns: 'client' }) }}
+      </h3>
+      <input
+        onclick="this.select();"
+        class="lux-input"
+        type="text"
+        :value="shortUrl"
+        readonly="true"
+      />
+    </div>
+    <img class="mx-5 mt-5" v-if="qrUrl" :src="qrUrl" />
+  </div>
+  <div>
+    <div class="mt-5">
+      <h3 class="text-3xl text-white">
+        {{ t('Location Coordinates', { ns: 'client' }) }}
+      </h3>
+    </div>
+    <table>
+      <tbody class="lux-info-table">
+        <tr v-for="(coords, label) in formatted_coordinates" :key="label">
+          <th>{{ t(label as string) }}</th>
+          <td>{{ coords }}</td>
+        </tr>
+        <tr>
+          <th style="text-align: left">{{ t('Elevation') }}</th>
+          <td>{{ elevation }}</td>
+        </tr>
+        <tr>
+          <th style="text-align: left">{{ t('Adresse la plus proche') }}</th>
+          <td>{{ address?.formattedAddress }}</td>
+        </tr>
+        <tr>
+          <th style="text-align: left">{{ t('Distance approximative') }}</th>
+          <td>{{ address?.distance }}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div>
+    <div v-if="isRapportForageVirtuelAvailable">
```

Comment:
look like this div is not useful and the condition can be attached to the button below

Reply:
removesd